src/family/dto/create-family.input.ts

```diff
@@ -0,0 +1,7 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateFamilyInput {
+  @Field()
+  name: string;
+}
```

Comment:
it's better moved into dto folder

`dto/input/create-family-input.dto.ts`

Reply:
Done